projects/plugins/jetpack/extensions/blocks/payments-intro/edit.js

```diff
@@ -38,15 +36,20 @@ export default function JetpackPaymentsIntroEdit( { name, clientId } ) {
 
 	const renderVariationPicker = () => {
 		return (
-			<BlockVariationPicker
+			<Placeholder
 				icon={ get( blockType, [ 'icon', 'src' ] ) }
 				label={ get( blockType, [ 'title' ] ) }
 				instructions={ __( "Please select which kind of payment you'd like to add.", 'jetpack' ) }
-				variations={ usableVariations }
-				onSelect={ ( nextVariation = usableVariations[ 0 ] ) => {
-					setVariation( nextVariation );
-				} }
-			/>
+				className="wp-payments-intro-wrapper"
```

Comment:
I _think_ (but can't check ATM) we could use the auto-generated className which should be `.wp-block-jetpack-payments-intro` by simply adding the className prop:

```jsx
export default function JetpackPaymentsIntroEdit( { name, clientId, className } ) {
  // ...
  <Placeholder className={ className } />
}
```